slurm_snakefile

```diff
@@ -0,0 +1,685 @@
+import itertools
+
+DATASETS, = glob_wildcards("dataset_configs/{dataset}.yml")
+
+SUPERVISED, = glob_wildcards("model_configs/supervised/{supervised}.yml")
+IMPUTE, = glob_wildcards("model_configs/imputation/{impute}.yml")
+UNSUPERVISED, = glob_wildcards("model_configs/unsupervised/{unsupervised}.yml")
+SEMISUPERVISED, = glob_wildcards("model_configs/semi-supervised/{semisupervised}.yml")
+
+NUM_SEEDS = 3
+
+recount_top_five_tissues = ['Blood', 'Breast', 'Stem_Cell', 'Cervix', 'Brain']
+combo_iterator = itertools.combinations(recount_top_five_tissues, 2)
+RECOUNT_TISSUE_STRING = ['.'.join(pair) for pair in combo_iterator]
+
+gtex_top_five_tissues = ['Blood', 'Brain', 'Skin', 'Esophagus', 'Blood_Vessel']
+combo_iterator = itertools.combinations(gtex_top_five_tissues, 2)
+GTEX_TISSUE_STRING = ['.'.join(pair) for pair in combo_iterator]
+
+TCGA_GENES = ['EGFR', 'IDH1', 'KRAS', 'PIK3CA', 'SETD2', 'TP53']
+
+wildcard_constraints:
+    # Random seeds should be numbers
+    seed="\d+"
+
+rule all:
+    input:
+        "data/batch_sim_data.tsv",
+        "data/linear_batch_sim_data.tsv",
+        "data/no_signal_batch_sim_data.tsv",
+	# Recount Binary classification
+        expand("/scratch/summit/benheil@xsede.org/results/{tissues}.{supervised}_{seed}.tsv",
```

Comment:
Could you set this directory `/scratch/summit/benheil@xsede.org/results/` in a variable at the top of your script or something like that, since it's used throughout the script? Or are you fairly confident it'll never need to be changed?